content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md

```diff
@@ -86,19 +98,19 @@ Instructions for setting up the Docker repository for your respective Linux dist
    ```
 
 <!--
-3. Restart containerd:
```

Comment:
Why we change "3" to "1" ?

Reply:
This is the original content of English upstream.